policy.md

```diff
@@ -196,18 +137,53 @@ Where splitter must be one of the following:
 * \-
 * \~
 
+### 5.2 Auditor Qualifications ###
+
+CA Owners must submit a summary of the Audit Team's qualifications and experience as outlined below with respect to the audit. The information can also be provided as part of the audit result documentation, like the [ETSI AAL](https://www.acab-c.com/downloads/), or as a supplement to the WebTrust Assurance Report.
+
+* Date that the audit report was signed,
+* Full name of the CA that was audited,
+* Name and address of the audit firm or CAB,
+* Audit Criteria, e.g. ETSI / WebTrust,
+* Name of Lead Auditor (except where prohibited by law, otherwise, we ask that you not provide any personally identifiable information),
+* For the Audit Team and the Audit Quality Reviewer, qualification information such as:
+    * Number of Audit Team Members,
+    * Academic qualifications or professional training received,
+    * Average Years of Auditing Experience auditing trust services or similar information systems,
+    * Experience, Special Skills, and Qualifications (e.g. audit/assessment principles and functions, information technology, software development, trust services, public key infrastructure, CA operations, and information security including risk assessment/management, network security, physical security, etc.), and
+    * Credentials, Designations, or Certifications (e.g. CPA, CISA, CITP, CISSP, CCSP/CCA/CCP, etc.).
+* How the Audit Team and team members are bound by law, regulation or professional standards to render an independent assessment of the CA (e.g.https://pub.aicpa.org/codeofconduct/Ethics.aspx# 0.300.050 Objectivity and Independence; CPA Canada, Rule 204; or Annex A of ETSI EN 319 403/403-1, respectively),
+* Name of the Insurance Carrier providing the professional liability or errors and omissions insurance coverage defined in CA/B Forum Baseline Requirements section 8.2., and
```

Comment:
I have deleted this bullet item from the Mozilla wiki guidance - https://wiki.mozilla.org/CA/Audit_Statements#Providing_Auditor_Qualifications because it was beyond what was necessary to determine auditor qualifications in the areas of independence, skill, etc.

Reply:
Thanks, Ben. I will remove from here as well. 

Also, I will add `Proof of audit firm or CAB Accreditation (URL)` which is present in the Mozilla wiki guidance but absent from here.